src/jade/public/index.jade

```diff
@@ -226,6 +226,17 @@ block content
                                 i.fa.fa-github.fa-fw
                                 span.network-name
                                     | Visit the Github project page
+     section#about.slide
+        .download-section
+            .container
+                .col-lg-8.col-lg-offset-2
+                    h1.animate-in(data-animation="space-in" data-build="1")
+                        | About the Guake Project
+
+                    p.animate-build(data-animation="fade-in" data-build="2")
+                        | Guake was originally written by Gabriel Falcão in 2007, but has since become
```

Comment:
Probably replace "but" with "and"?  It has a been "feel" to it that wau. :smile: